packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/delete/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dedicatedCloud_datacenter_backup_delete_title": "Désactivation de la sauvegarde",
+  "dedicatedCloud_datacenter_backup_delete_description": "Souhaitez-vous désactiver la sauvegarde pour le datacenter {{datacenter}} ? Vous devrez vérifier la désactivation de la sauvegarde pour chaque machine virtuelle depuis vSphere. Attention, cette action supprimera définitivement toutes vos sauvegardes.",
```

Comment:
"pour le datacentre {...}"

Reply:
I did not understand this comment. Can you please elaborate on it.

Reply:
```suggestion
  "dedicatedCloud_datacenter_backup_delete_description": "Souhaitez-vous désactiver la sauvegarde pour le datacentre {{datacenter}} ? Vous devrez vérifier la désactivation de la sauvegarde pour chaque machine virtuelle depuis vSphere. Attention, cette action supprimera définitivement toutes vos sauvegardes.",
```